ddtrace/internal/datastreams/processor.py

```diff
@@ -65,6 +66,8 @@ def gzip_compress(payload):
 log = get_logger(__name__)
 
 PROPAGATION_KEY = "dd-pathway-ctx"
+PROPAGATION_KEY_BASE_64 = "dd-pathway-ctx-base64"
```

Comment:
nit: This constant is not used in this file. Is there a better place to put this?